src/helpers/_focusable.scss

```diff
@@ -30,5 +29,28 @@
     outline: $govuk-focus-width solid $govuk-focus-colour;
     outline-offset: 0;
     background-color: $govuk-focus-colour;
+    text-decoration: none;
```

Comment:
@dashouse can you confirm that you want to remove text-decoration here? It's think it's only used in the skip-link component after these changes, which I'm not sure is a good idea since the underline helps give it the affordance that it can be clicked?

We want to review these mixins once we've done all the work but just wondering here...

Reply:
This should only affect links that are focussed, I think? I believe the intention is to avoid a 'double underline' from the text decoration and the black box shadow together.

Reply:
This mixin is no longer used with links, I think you're replying thinking of the mixin below this one potentially?

Reply:
Spoke to Dave and given that this is only used by tabbing to the component we've decided to leave the underline in for now.